design-library/src/components/BccSelectionElement/BccSelectionElement.vue

```diff
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+import { ArrowForwardIcon } from "@bcc-code/icons-vue";
+type Props = {
+  isLast?: boolean;
```

Comment:
Instead of doing it like this, another option would be to create a wrapping component that will render the borders. Something like:

```vue
<BccSelectionList>
   <BccSelectionListItem ... />
   <BccSelectionListItem ... />
   <BccSelectionListItem ... /> 
</BccSelectionList>
```

I would have a personal preference for that as I think it's a bit cleaner, and would also simplify usage.